recipes/qr-code-generator/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake"
```

Not required for your test package, as you are using CONAN_LIBS variable.